CoinDock-api/app/Models/V1/User.php

```diff
@@ -100,4 +450,8 @@ public function signUp()
     {
         return $this->hasOne(Signup::class);
     }
+
+    public function wallet(){
```

Comment:
if the relation was hasMany() then the function name should be plural.

Reply:
Modified